Wangscape/noise/module/ModuleFactories.cpp

```diff
@@ -0,0 +1,109 @@
+#include "ModuleFactories.h"
+#include "Gradient.h"
+#include "NormLPQ.h"
+#include "../MakeReseedable.h"
+#include <stdexcept>
+
+namespace noise
+{
+namespace module
+{
+
+Reseedable makeQuadrantSelector(Reseedable& source, bool x_positive, bool y_positive)
+{
+    if (x_positive && y_positive)
```

Comment:
This `if` is redundant and a bit misleading. Without it this method will work the same way, if I see correctly.